gen/calc.py

```diff
@@ -371,6 +371,7 @@ def validate_os_type(os_type):
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
         'ui_banner_dismissible': 'null',
+        'dcos_overlay_config_attempts': '4',
```

Comment:
I will put an integer check and range check of 0 to 10. We have exponential back off in place when the configuration fails, so if go beyond 10 tries it pretty much de-generates the fix into the original behavior which no hard stop on failed configuration.